### ISSUE TITLE
Share Instagram and Facebook does not work on Android 11 S21 Ultra

### ISSUE THREAD

**Issue by armandojimenez** [ShekarMudaliyar/social_share#67]
Im trying to share to Facebook and IG on an Android 11 Samsung Galaxy S21 Ultra, but it does not work, works on other Android devices. Im running my project onAndroid SDK 30. The return error is simply 'error'...

**Comment by armandojimenez** [comment]
Fixed it by adding to my manifest:

<queries>
        <intent>
            <action android:name="android.intent.action.SEND" />
            <data android:mimeType="image/*" />
        </intent>
    </queries>

**Comment by lpancho** [comment]
Yes that's the fix but the missing items are the packages of the app you want to interact with. See full queries tag below

`<queries>
        <package android:name="com.facebook.katana"/>
        <package android:name="com.instagram.android"/>
        <intent>
            <action android:name="android.intent.action.SEND" />
            <data android:mimeType="image/*" />
        </intent>
    </queries>`

**Comment by armandojimenez** [comment]
Thanks! added the complete query

**Comment by gulmensedat** [comment]
> 

I can't use it as you said, I can use it this way, but it doesn't work, can you share the full code of androidmanifest?

i use
`    <intent-filter>
                <action android:name="android.intent.action.SEND" />
                <data android:mimeType="image/*" />
            </intent-filter>`

**Comment by coderjenil** [comment]
thanks
